### ISSUE TITLE
Link mismatch on Events page

### ISSUE THREAD

**Issue by debo19** [layer5io/layer5#3399]
#### Description

Due to pagination, the url changes on moving to older events. Due to this, clicking on the upcoming events from the carousal appends the new event address to this new url, resulting in a 404 error.

https://user-images.githubusercontent.com/85789734/197029228-814b193d-9e37-4147-ae6e-419ca275822b.mov

#### Expected Behavior
The link should direct to that particular event on click.

#### Environment:
- Host OS: 
- Browser: 

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="16px" align="left" /><h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by AbhishekTiwari23** [comment]
Do you want them directed to this right[page](https://layer5.io/community/events/kubecon-cloudnative-north-america-2022) ?

**Comment by debo19** [comment]
@AbhishekTiwari23 Right, yes. Not only this event but the rest of them as well (that are visited through the carousal).

**Comment by dishantzaveri** [comment]
pls assign it to me

**Comment by legendarykamal** [comment]
@dishantzaveri I don't see but will you please confirm in if you have your project setup up and running ? as it is neccessary for the task. 

Please remove your assignment if not , take some time to do the setup , come back and take charge 💯

**Comment by UtkarshMishra12** [comment]
@dishantzaveri Please update

**Comment by Oghene-kevwe** [comment]
Hello, I'd like to work on this

**Comment by UtkarshMishra12** [comment]
Here you go @Oghene-kevwe 🚀

**Comment by UtkarshMishra12** [comment]
@Oghene-kevwe Please Update

**Comment by Oghene-kevwe** [comment]
@UtkarshMishra12  I'm still on it

**Comment by Oghene-kevwe** [comment]
@debo19 @leecalcote  I have fixed this but it hasn't been merged

**Comment by Nikhil-Ladha** [comment]
Ahh....ok
That's the reason you should always have a practice of mentioning the issue number in the PR description. Please update the same.

**Comment by Oghene-kevwe** [comment]
@Nikhil-Ladha Thank you, I'll do that now
